### PR TITLE
EREGCSC-1335 Synonym message for search

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # 1335

Adds the search for synonym message for the resources page

- expected change 1

**Steps to manually verify this change...**

1. steps to view and verify change

Search for a word with a synonym.  example ELE

A message should appear  with "Search for similar terms: {{then the list of synonyms}} seperated by commas

Click on one of them.  It will update the search box and the list of synonyms.  If it is multiple words Didn't find what you were looking for? Try searching for will appear as well as "Or" befoe "Search for similar term:"

### CODE REVIEW

File: solution/ui/regulations/eregs-vite/src/views/Resources.vue [hunk 1]
```diff
@@ -197,6 +210,17 @@ export default {
                 },
             });
         },
+        async synonymLinks(synonym){
```
Comment (knollfear) [inline solution/ui/regulations/eregs-vite/src/views/Resources.vue:213]: Is this needed?  won't it just happen for free when you re-execute search?

File: solution/ui/regulations/eregs-vite/src/views/Resources.vue [hunk 2]
```diff
@@ -21,11 +21,21 @@
...
-                      <a @click="doQuoteSearch">"{{this.searchQuery}}"</a>
+                    <div class="search-suggestion">
+                        <div v-if="multiWordQuery">
+                            Didn't find what you were looking for? Try searching for
+                            <a @click="doQuoteSearch">"{{this.searchQuery}}"</a>
+                        </div>
+                        <div class="synonyms" v-if="synonyms.length > 0"> 
+                            <span v-if="multiWordQuery">Or </span>Search for similar terms:
+                            <span v-bind:key=a v-for="a in synonyms">
+                                "<a @click="synonymLinks(a)">{{a}}</a>"
```
Comment (knollfear) [inline solution/ui/regulations/eregs-vite/src/views/Resources.vue:32]: I think no quotes around the synonym links per Britta's comment in slack.

**Comment by PhilR8** [comment]
we've lost some margin below the search box:
Screen Shot 2022-07-28 at 13 16 26

Compare to prod, there should be about 50px of margin between the bottom of the search box and the bottom of the blue container

File: solution/ui/regulations/eregs-vite/src/views/Resources.vue (hunk 2 above)
Comment (thwalker6) [reply solution/ui/regulations/eregs-vite/src/views/Resources.vue:32]: Removed

File: solution/ui/regulations/eregs-vite/src/views/Resources.vue (hunk 1 above)
Comment (thwalker6) [reply solution/ui/regulations/eregs-vite/src/views/Resources.vue:241]: It is but I refactored to use most of execute searches code

**Comment by thwalker6** [comment]
> we've lost some margin below the search box: Screen Shot 2022-07-28 at 13 16 26
> 
> Compare to prod, there should be about 50px of margin between the bottom of the search box and the bottom of the blue container

Fixed the css issue.

**Review by knollfear — APPROVED** [review]
LGTM!.  The margin seems to be back and looks great.

**Pull request merged**